sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/XrayWrapperException.java

```diff
@@ -0,0 +1,22 @@
+package com.mercedesbenz.sechub.wrapper.xray;
+
+import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperExitCode;
+
+public class XrayWrapperException extends Exception {
+
```

Comment:
To avoid warnings about missing serial version uids, please add


```suggestion
private static final long serialVersionUID = 1L, 
```